lib/base/scriptframe.cpp

```diff
@@ -38,11 +38,11 @@ INITIALIZE_ONCE_WITH_PRIORITY([]() {
 
 	Namespace::Ptr internalNS = new Namespace(l_InternalNSBehavior);
 	globalNS->SetAttribute("Internal", new ConstEmbeddedNamespaceValue(internalNS));
-}, 1000);
+}, InitializePriority::CreateNamespaces);
 
 INITIALIZE_ONCE_WITH_PRIORITY([]() {
 	l_InternalNSBehavior->Freeze();
-}, 0);
+}, InitializePriority::Default);
```

Comment:
Why not to omit the prio altogether in such cases?

Reply:
That's a question for the original author. For now I kept it like this because #9607 would have [changed](https://github.com/Icinga/icinga2/pull/9607/files#diff-496fd29af09e81a2cb2c3b0faab08cc52aebb05249a70933e3cf4f999a2e0b10L43-R51) this to another priority anyways. Now that #9607 is [in limbo](https://github.com/Icinga/icinga2/pull/9607#issuecomment-1351494685), let's see what we do here.

Reply:
Yes...

https://github.com/Icinga/icinga2/commit/8fda8d72acb5c96565665903a0b6116dff5680ba#diff-496fd29af09e81a2cb2c3b0faab08cc52aebb05249a70933e3cf4f999a2e0b10R53-R55

... but now with the names and the obvious knowledge what's that magic 0 it looks even more strange. I'd change it unless #9607 will be merged.